packages/core/src/pagination/pagination.element.scss

```diff
@@ -0,0 +1,11 @@
+@import './../styles/tokens/generated/index';
+
+:host {
+  --background: none;
+  --color: #{$cds-global-typography-color-500};
+  --font-size: #{$cds-global-typography-font-size-4};
+  --disabled-color: #{$cds-alias-status-disabled};
```

Comment:
I see a number of custom properties here but it's not clear where or if they are used. I see `--color` being used on line 9. What about the others?

Reply:
Removed most of the css properties except `--color` and `--font-size`.